great_expectations/expectations/core/expect_select_column_values_to_be_unique_within_record.py

```diff
@@ -24,16 +24,12 @@ class ExpectSelectColumnValuesToBeUniqueWithinRecord(ColumnMapExpectation):
         "requirements": [],
     }
 
-    metric_dependencies = tuple()
-    success_keys = (
-        "column_list",
-        "ignore_row_if",
-        "mostly",
-    )
+    map_metric = "select_column_values.unique.within_record"
+    success_keys = ("sum_total",)
```

Comment:
Why are `success_keys` so different than in #3234, for example? I'm not sure I understand how this is being used.

Reply:
@NathanFarmer This was a typo -- nice catch (fixed).  Thank you!